services/backend/src/database/migrations/20191031145350-create-user-and-profile.js

```diff
@@ -140,6 +140,7 @@ module.exports = {
               careerInterests: true,
               mentorshipSkills: true,
               exFeeder: true,
+              security: false,
```

Comment:
Would need to add a new field to the `schema.prisma` file in the database, then create a new migrate based on the prisma cli or by doing `yarn migrate:create`, then apply the migration with `yarn migrate`, and probably also update the seeds file under database > seed > fake profile/user data